### DISCUSSION TITLE
Name of the Alumni

### DISCUSSION THREAD

**Discussion by bipon68** [Bipon-Dev/kpi-alumni-frontend#14] [Q&A] [answered]
deshialumni.com
alumnicorners.com
PolytechnicHub.com
AlumniConnectBD.com
AlumniConnect.com
Polytechnic Professionals Network (PPN)
Polytechnic Alumni Unity Bangladesh (PAUB)
Bangladesh Polytechnic Alumni Association (BPAA)

**Comment by panthosarkar** [comment, marked as answer]
Bangladesh Polytechnic Alumni Association (BPAA)
